### DISCUSSION TITLE
Cannot connect to remote MYSQL database : Linux Installation

### DISCUSSION THREAD

**Discussion by JuliRash** [requarks/wiki#5288] [Help / Questions]
Hello folks, I have tried to set up wiki js using the Linux installation guide and I have tried my best to connect to a remote database but it keeps failing and seeing the database host as localhost after changing it to the remote database IP address. please help 

```2022-05-14T05:42:56.817Z [MASTER] info: Connecting to database...
2022-05-14T05:42:56.826Z [MASTER] error: Database Connection Error: ER_DBACCESS_DENIED_ERROR undefined:undefined```

**Comment by patrick-othmer** [comment]
What solution do you use? Docker or bare metal?
ER_DBACCESS_DENIED_ERROR -> looks like a access issues so do you checked that the mysql server is available from your wiki.js server? Did the mysql user have access from the remote host (Hostname at the mysql permissions)? wiki.js in Docker with a remote MariaDB cluster is no problem on my setup (beside some connection timeout issues after some time - looks like a problem with the mysql2 database lib.).
